src/cdk/a11y/a11y.md

```diff
@@ -55,6 +55,67 @@ interface Highlightable extends ListKeyManagerOption {
 
 Each item must also have an ID bound to the listbox's or menu's `aria-activedescendant`.
 
+### TreeKeyManager
+
+`TreeKeyManager` manages the active option in a tree view. Use this key manager for
+components that implement a `role="tree"` pattern.
+
+#### Basic usage
+
+Any component that uses a `TreeKeyManager` should do three things:
+* Create a `@ViewChildren` query for the tree items being managed.
+* Initialize the `TreeKeyManager`, passing in the options.
+* Forward keyboard events from the managed component to the `TreeKeyManager` via `onKeydown`.
+
+Each tree item should implement the `TreeKeyManagerItem` interface:
```

Comment:
Ideally this links to the API docs so that we don't need to keep this in sync, e.g. https://v11.material.angular.io/cdk/a11y/api#TreeKeyManagerItem